activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java

```diff
@@ -19,13 +19,13 @@
 import javax.jms.JMSException;
 import javax.jms.QueueSender;
 import org.apache.activemq.ActiveMQConnectionFactory;
-import zipkin2.Call;
-import zipkin2.Callback;
-import zipkin2.CheckResult;
-import zipkin2.codec.Encoding;
 import zipkin2.reporter.AsyncReporter;
```

Comment:
IIUC this import is only used in javadoc. For similar lines, just a nit but you may want to inline it instead to make it easier to scan the imports for usage of the zipkin-tied types, or lack of

Reply:
good idea. I'll look around for this!